.kitchen-docker.yml

```diff
@@ -0,0 +1,75 @@
+driver:
+  name: dokken
+  chef_version: latest
+  privileged: true # because Docker and SystemD/Upstart
+
+transport:
+  name: dokken
+
+provisioner:
+  name: dokken
+
+verifier:
+  name: inspec
+
+platforms:
+- name: debian-7
```

Comment:
I do not think we should put any effort into debian-7 as it is EOL, I think if anything we should support 8 and 9
https://www.debian.org/releases/